test/lib/ufe/testLight.py

```diff
@@ -251,6 +279,38 @@ def testUsdLight(self):
         usdAreaLight = usdUtils.getPrimFromSceneItem(arealightItem)
         self._TestAreaLight(ufeAreaLight, usdAreaLight)
 
+    @unittest.skipUnless(os.getenv('UFE_VOLUME_LIGHTS_SUPPORT', 'FALSE') == 'TRUE', 'UFE has volume light support.')
+    def testUsdLight_v5_5(self):
+        self._StartTest('SimpleLight')
+        mayaPathSegment = mayaUtils.createUfePathSegment('|stage|stageShape')
+        # test cylinder light
+        cylinderlightUsdPathSegment = usdUtils.createUfePathSegment('/lights/cylinderLight')
+        cylinderlightPath = ufe.Path([mayaPathSegment, cylinderlightUsdPathSegment])
+        cylinderlightItem = ufe.Hierarchy.createItem(cylinderlightPath)
+
+        light_v5_5 = ufe.Light_v5_5()
```

Comment:
This should be using the hasattr to know whether we need to use the Light_v5_5 or just the regular Light.

Reply:
It's under `skipUnless(os.getenv('UFE_VOLUME_LIGHTS_SUPPORT'` so it should already have the Light_v5_5 support